### ISSUE TITLE
Auto generate barcode

### ISSUE THREAD

**Issue by rudvolkov** [allegro/ralph#3532]
Does the system have a barcode generation function?

**Comment by romcheg** [comment]
@rudvolkov No, the barcodes should be entered manually. The only option for automating the process was using @vi4m 's iphone app but I'm not sure it still exists in AppStore.

**Comment by rudvolkov** [comment]
Will the barcode auto-generation function be added in new versions?

**Comment by romcheg** [comment]
Unfortunately we did not have it in our plans. However, we will gladly accept a change introducing that functionality.

**Comment by rudvolkov** [comment]
@romcheg I used to see the order in 2017, they also asked about barcode, there was an answer that in the future they might introduce a system, so I decided to clarify.
Ник скрывает русскую натуру?)

**Comment by romcheg** [comment]
Due to limited capacity of the maintenance team we have to prioritise the most essential changes and cut off the rest. The prioritisation on our case has to be done basing on how we use Ralph internally. In our scenario barcodes are generated before assets are introduced into Ralph so it's unlikely we will be be working on auto-generating them in observable future.

Open-source software gives everyone an opportunity to change or add whatever they need. Our approach of making Ralph welcomes contribution from anyone and supports contributors who need some support. To be frank with everyone I have to clarify this -- this feature would be useful for many users yet it will have the best chance of being added to Ralph, if it is done by someone but the maintenance team.

P. S. No, I am not Russian :)
